### PR TITLE
feat: Allow showing territory owner [skip ci]

### PR DESCRIPTION
**Pull request by cmkohnen**
Yet another small change allowing to show the current territory's owner in the territory overlay, matching the Info Bar.

### CODE REVIEW

**Comment by ShadowCat117** [comment]
You might want to change the function to handle when you are not inside a territory, otherwise you'll end up with `[]`

**Comment by ShadowCat117** [comment]
I also wonder if it would be worth adding a config to show the full guild name instead, maybe then put the name on a separate line

**Comment by cmkohnen** [comment]
> You might want to change the function to handle when you are not inside a territory, otherwise you'll end up with `[]`

oops, thanks for noticing.

The three options now are:
Bildschirmfoto vom 2024-05-23 18-21-43
Bildschirmfoto vom 2024-05-23 18-21-59
Bildschirmfoto vom 2024-05-23 18-22-06

With just the territory being the default still

File: common/src/main/java/com/wynntils/overlays/minimap/TerritoryOverlay.java [hunk 1]
```diff
@@ -22,18 +25,34 @@ public TerritoryOverlay() {
...
+        return showOwner.get().getTemplate();
     }
 
     @Override
     public String getPreviewTemplate() {
-        return TEMPLATE;
+        return getTemplate();
+    }
+
+    public enum TerritoryOwnerDisplay {
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/overlays/minimap/TerritoryOverlay.java]:
```suggestion
    private enum TerritoryOwnerDisplay {
```

**Review by kristofbolyai — APPROVED** [review]
Looks good, although I wouldn't add too many of these, I think we should provide sensible defaults, then everyone can customize their UI to their needs. Either way, extending this overlay to have the guild owner is a nice touch.

**Comment by ShadowCat117** [comment]
> Looks good, although I wouldn't add too many of these, I think we should provide sensible defaults, then everyone can customize their UI to their needs.

I think these 3 options are nice, if someone wants both name and tag or for it to be coloured when the owner matches their guild then that should be in a custom overlay

**Pull request merged**